### ISSUE TITLE
Error in msd_mutate: unused argument (primer_length = input$ms_primer_length)

### ISSUE THREAD

**Issue by sneumann** [ipb-halle/GoldenMutagenesisWeb#6]
Hi, when running with sequence `ATGAAAGGGCCCTTTTAA` and position=2
and saturation="NNK", we don't reach the primer design stage:
```
Listening on http://127.0.0.1:7346
[1] "pAGM9121 selected"
[1] "2"   "NNK"

Warning: Error in msd_mutate: unused argument (primer_length = input$ms_primer_length)
  72: observeEventHandler [/home/sneumann/src/R/GoldenMutagenesisWeb/app.R#976]
```
Yours, Steffen

**Comment by culpinnis** [comment]
Was the gene sequence somehow shortened during the copy/paste?
The sequence is way too short for a protein, even smaller than one single primer. I agree, that there should be a useful error message (and a check) for too short sequences.

**Comment by culpinnis** [comment]
General error in mutate_msd/spm is fixed in 9dad9b2 (compatibility to GM Version 1.1.0)
